public/index.html

```diff
@@ -0,0 +1,370 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>W2-A2-Benton-Mark</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+    <a id="home"></a>
+    <nav class="drop-down">
+        <ul class="flex-container nav-ul">
+            <li class="selected nav-li"><a href="#home">Home.</a>
+            <li class="hidden-class nav-li"><a href="#services">Services.</a></li>
+            <li class="hidden-class nav-li"><a href="#portfolio">Portfolio.</a></li>
+            <li class="hidden-class nav-li"><a href="#about">About.</a></li>
+            <li class="hidden-class nav-li"><a href="#team">Team.</a></li>
+            <li class="hidden-class nav-li"><a href="#contact">Contact.</a></li>
+        </ul>
+    </nav>
+    <section class="header-section">
+        <header>
+            <img src="./imgs/top-logo.gif" alt="main page logo">
+            <p>Hi there! We're Avengers, <strong>creative design agency</strong>
+                <br>we build <span class="awesome-sm"><strong>awesome</strong></span> mobile and websites</p>
+            <object class="awesome" type="image/svg+xml" data="./imgs/underline.svg"></object>
+        </header>
+    </section>
+    <section class="section-orange">
+        <object class="clip clip-white" type="image/svg+xml" data="./imgs/clipwhite.svg">
+        </object>
```

Comment:
Again with the `object`. Prefer `img`, inline `svg`, or `background-image`.

Also, since this element has no content, the closing tag should be on the same line as the opening tag.